rmgpy/data/rmg.py

```diff
@@ -62,15 +63,12 @@ def __init__(self):
         self.kinetics = None
         self.statmech = None
         self.solvation = None
-        
+
         # Store the newly created database in the module.
         global database
-#        assert database is None, "Should only make one instance of RMGDatabase because it's stored as a module-level variable."
-        if database is None:
-            database = self
-        else:
-            logging.warning("Should only make one instance of RMGDatabase because it's stored as a module-level variable!")
-            logging.warning("Unexpected behaviour may result!")
+        if database is not None:
+            logging.warn('An instance of RMGDatabase already exists. Re-initializing it.')
```

Comment:
I think we should use `logging.warning`. `logging.warn` is just a pointer to `logging.warning`, and is now deprecated in Python 3.